recipe/meta.yaml

```
@@ -26,7 +26,6 @@ requirements:
     - {{ compiler('cxx') }}
     - sysroot_linux-64 2.17  # [linux64]
   host:
-    - arrow-cpp >=10.0
     - blosc
```

For the global build requirements, you cannot switch yet, assuming you are still going to be building against previous versions of arrow.

Either you stay on arrow-cpp for everything here for a while longer (we'll keep that output working), or you do something as follows:

```suggestion
    # libarrow only exists from 10.x
    - arrow-cpp  # [arrow_cpp in ("6.0.2", "7.0.1", "8.0.1", "9.0.0")]
    - blosc
```